PWAWidgets/README.md

```diff
@@ -232,6 +233,7 @@ One or more Widgets are defined within the `widgets` member of a Web App Manifes
 ### Optional business logic properties
 
 * `auth` - Boolean. Informational. Whether or not the Widget requires auth. False if not included.
+* `prevent_multiple` - Boolean. Whether or not the multiple instances of this widget are allowed. False if not included.
```

Comment:
```suggestion
* `prevent_multiple` - Boolean. Whether or not the multiple instances of this widget are allowed, on a per widget host basis. False if not included.
```

There will likely be a multi-host future, so we should probably clarify this early.